MC-1700 Docs for Index stats table (#195)
hazelcast/management-center-docs: docs/modules/data-structures/pages/map.adoc

To ascend or descend the order of the listings, click on the
column headings.

[[map-index-stats]]
The **Map Index Statistics** table provides statistics for each index in a map.

NOTE: This table is displayed only if the map has at least one index. To learn how to create indexes, see xref:{page-latest-supported-hazelcast}@hazelcast:query:indexing-maps.adoc[Indexing Maps] in the Hazelcast Platform documentation.

image:ROOT:MapIndexStatsTable.png[Map Index Stats Table]

You can see member-level statistics for each index by clicking on the down arrow next to the index name.

From left to right this table lists the following:

.Map index statistics for the cluster
[cols="20%s,80%a"]
|===
|Field|Description

|Index Name
|Name of the index which was created for this map.

|Creation Time

> |Creation time of the index for the first time among the members.

Reviewer: ```suggestion
|The first time that the index was created on a member in the cluster.
```

To ascend or descend the order of the listings, click on the
column headings.

[[map-index-stats]]

> **Map Index Stats Table** provides information about Map Index statistics if there exists any index for this map.

Reviewer 2: `if there exists any index` - is this grammatically correct?

Author: you are right `if there is any `  would be better 👍

> NOTE: You need to create indexes to see this table here. See the xref:{page-latest-supported-hazelcast}@hazelcast:query:indexing-maps.adoc[Indexing Maps] for details.

Reviewer: ```suggestion
NOTE: This table is displayed only if the map has at least one index. To learn how to create indexes, see xref:{page-latest-supported-hazelcast}@hazelcast:query:indexing-maps.adoc)[Indexing Maps] in the Hazelcast Platform documentation.
```

> NOTE: You need to create index to see this table here. See the xref:{page-latest-supported-hazelcast}@hazelcast:query:indexing-maps.adoc[Indexing Maps] for details

Reviewer 3: ... create indexes to see ...   ... for details.

> image:ROOT:MapIndexStatsTable.png[Map Index Stats Table]

Reviewer 2: I don't think we should post two images to show all the columns in the table.
I think a single image will be enough.
@JakeSCahill WDYT?

Reviewer: I think we don't even need a screenshot here. We provide the information in the table.

Author: I think we need it at least one of them. it's nice to see the table when you are reading the provided information and in addition to that all other tables have their own screenshots. We should follow the pattern, imho.

image:ROOT:MapIndexStatsTable2.png[Map Index Stats Table-2]

> To see more details about an index, click on the down arrow next to the index name.

Reviewer 3: Let's remove this line, and rephrase it as suggested in the next review comment below.

> You can see more details about an index by clicking on the down arrow next to the index name.

Reviewer: ```suggestion
You can see member-level statistics for each index by clicking on the down arrow next to the index name.
```

From left to right this table lists the following:

> .Map Index Stats Table Fields

Reviewer: ```suggestion
.Top-level fields for map index statistics
```

[cols="20%s,80%a"]
|===
|Field|Description

|Index Name
|Name of the index which was created for this map.

|Creation Time

> |Creation time of the index for the first time among the members.

Reviewer: How is this different compared to the member-level view?

Author: It's the smallest creation time among the members. In the member-level, it is creation time of index in that member.

|Total Memory Cost
|Total local memory cost of the index.

|Total Query Count
|Total number of queries served by the index.

|Total Insert Count
|Total number of insert operations performed on the index.

|Average Insert Time
|Average time of insert operations performed on the index among the members.

|Total Update Count
|Total number of update operations performed on the index.

|Average Update Time
|Average time of update operations performed on the index among the members.

|Total Remove Count
|Total number of remove operations performed on the index.

|Average Remove Time
|Average time of remove operations performed on the index among the members.

|Total Hit Count
|Total number of index hits.

|Average Hit Time
|Average time of index hits performed on the index among the members.

|Average Hit Selectivity
|Average selectivity of the hits served by the index among the members.
|===

> 

Reviewer 3: Maybe it is better to insert the "Table-2" image here, with a text as follows:

You can see more details about an index by clicking on the down arrow next to the index name:

((Table-2 image))

From left to right this table lists the following:

Author: These two table images belong to the same table. Couldn't be able to take a screenshot of all columns of the map index stats table because it has 13 columns. I divided it two images to display all the columns.

> .Map Index Stats Table Detailed Sub Row Fields

Reviewer: ```suggestion
.Second-level fields for map index statistics
```
